### ISSUE TITLE
warnings in assemblyinfo?

### ISSUE THREAD

**Issue by PandaWood** [mxriverlynn/Albacore#164]
I get these 3 warnings for every assemblyinfo file I create. Is it just me?

C:/Projects/build/bundler/ruby/1.9.1/gems/albacore-0.2.5/lib/albacore/assemblyinfo.rb:29: warning: instance variable @input_file no
t initialized
C:/Projects/build/bundler/ruby/1.9.1/gems/albacore-0.2.5/lib/albacore/assemblyinfo.rb:90: warning: instance variable @custom_attrib
utes not initialized
C:/Projects/build/bundler/ruby/1.9.1/gems/albacore-0.2.5/lib/albacore/assemblyinfo.rb:96: warning: instance variable @custom_data not initialized

I have 1.9.2 installed, even though it looks like I have 1.9.1 (see here for explanation of it's misleading http://stackoverflow.com/questions/6352262/why-does-bundler-install-gems-into-a-1-9-1-directory-when-1-9-2-is-the-only-ruby)

**Comment by mxriverlynn** [comment]
dont' think i've ever seen these warnings... what patch level of ruby 1.9.2 are you using?

**Comment by PandaWood** [comment]
I'm using p290
I was able to remove the warnings with a bit of work.
As it turns out, when it comes to this warning, the AssemblyInfo class is fairly fussy about what is set & when.
I had to set these variables in my calling code, explicitly, else I get a warning for each:

`lang_engine = CSharpEngine.new
custom_data = nil
input_file = nil`

Actually, I'm still getting a warning on @custom_attributes not initialized, in the cases where I don't explicitly set it so I'll add this to the list 
`custom_attributes = nil`

Apparently you don't get these warnings if you don't pass -W to Ruby. But I haven't done anything custom or fancy in that regard.
